CHANGELOG/CHANGELOG-1.25.md

```diff
@@ -1425,6 +1425,7 @@ ConfigMap rendering [issue](https://github.com/kubernetes/kubernetes/issues/1120
 - The beta `PodSecurityPolicy` admission plugin, deprecated since 1.21, is removed. Follow the instructions at https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/ to migrate to the built-in PodSecurity admission plugin (or to another third-party  policy webhook) prior to upgrading to v1.25. ([#109798](https://github.com/kubernetes/kubernetes/pull/109798), [@liggitt](https://github.com/liggitt))
 - VSphere releases less than 7.0u2 are not supported for in-tree vSphere volume  as of Kubernetes v1.25. Please consider upgrading vSphere (both ESXi and vCenter)  to 7.0u2 or above. ([#111255](https://github.com/kubernetes/kubernetes/pull/111255), [@divyenpatel](https://github.com/divyenpatel))
 - Windows winkernel kube-proxy no longer supports Windows HNS v1 APIs. ([#110957](https://github.com/kubernetes/kubernetes/pull/110957), [@papagalu](https://github.com/papagalu))
+- GlusterFS provisioner (`kubernetes.io/glusterfs`) has been deprecated in this release. ([#111485](https://github.com/kubernetes/kubernetes/pull/111485), [@humblec](https://github.com/humblec))
```

Comment:
Do you wanna remove this one since it has been moved to `### Deprecation` session: https://github.com/kubernetes/kubernetes/blob/7534d18847af7349dca0b98bb404645e282a0586/CHANGELOG/CHANGELOG-1.25.md?plain=1#L1671

And also this line: https://github.com/kubernetes/kubernetes/blob/7534d18847af7349dca0b98bb404645e282a0586/CHANGELOG/CHANGELOG-1.25.md?plain=1#L2150
may worth to move up in ### Deprecation as well?

Reply:
thanks done